msteams-platform/bots/how-to/conversations/conversation-messages.md

```diff
@@ -277,91 +277,16 @@ Messages received from or sent to your bot can include different types of messag
 | Cards     | ❌                | ✔️                | See the [Teams card reference](~/task-modules-and-cards/cards/cards-reference.md) for supported cards. |
 | Emojis    | ✔️                | ✔️                | Teams currently supports emojis through UTF-16, such as U+1F600 for grinning face. |
 
-## Notifications to your message
+## Types of messages
 
-You can also add notifications to your message using the `Notification.Alert` property. Notifications alert users about new tasks, mentions, and comments. These alerts are related to what users are working on or what they must look at by inserting a notice into their activity feed. For notifications to trigger from your bot message, set the `TeamsChannelData` objects `Notification.Alert` property to *true*. Whether or not a notification is raised depends on the individual user's Teams settings and you can't override these settings. The notification type is either a banner, or both a banner and an email.
+There are two different types of messages in bot conversation.
```

Comment:
This isn't correct, the table directly above this calls out different types of messages: https://review.learn.microsoft.com/en-us/microsoftteams/platform/bots/how-to/conversations/conversation-messages?branch=pr-en-us-6199&tabs=dotnet#message-content

And it includes emoji and rich text messages. And there are also other types of cards supported: https://review.learn.microsoft.com/en-us/microsoftteams/platform/task-modules-and-cards/cards/cards-reference?branch=pr-en-us-6199.

Reply:
Updated as mentioned.